spec/support/test_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Test
+  class TestController < ActionController::Base
+    use_openstax_exception_rescue
+
+    def bad_action
+      exception = params[:exception].constantize
+      raise case params[:exception]
+            when 'ActionController::RoutingError'
+              exception.new('routing error')
+            when 'Apipie::ParamMissing'
```

Comment:
Oh ya sure I guess I was just trying to keep it real but I can do that too, already have a test support file for that purpose